### PR TITLE
Updated README for java8 + eclipse juno

### PR DESCRIPTION
**Pull request by mswertz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,84 +1,68 @@
...
-    
-## 3. Install and configure eclipse
+install eclipse (we used Luna) from https://www.eclipse.org/downloads/?osType=macosx
 
-Compiling MOLGENIS requires Java 7.
-MOLGENIS is created with help of Maven and Freemarker. You need a few eclipse plugins to work with those.
+install mysql from (we used 5.6.23 DMG) http://dev.mysql.com/downloads/mysql/
 
-Important: get the LATEST version of Eclipse and all plugins mentioned. For example, using Eclipse Juno Release 1 or lower might result in infinite build loops.
+install git from http://git-scm.com/download/mac
```
Comment (fdlk) [inline README.md]: not all users will be mac users

Comment (mswertz) [inline README.md]: Can we test the protocol on a windows machine and edit appropriately?

On Tue, Mar 3, 2015 at 10:37 PM, Fleur Kelpin <notifications@github.com>
wrote:

> In README.md
> <https://github.com/molgenis/molgenis/pull/2561#discussion_r25729160>:
>
> >
> > -Important: get the LATEST version of Eclipse and all plugins mentioned. For example, using Eclipse Juno Release 1 or lower might result in infinite build loops.
> > +install git from http://git-scm.com/download/mac
>
> not all users will be mac users
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/molgenis/molgenis/pull/2561/files#r25729160>.
>



-- 
Morris A. Swertz
Associate Professor Big data in biomedicine
Lead BBMRI-NL common services for IT
Head Section Systems Genetics & Genomics Coordination Center
Dept of Genetics
University Medical Center Groningen
The Netherlands

File: README.md [hunk 2]
```diff
@@ -1,84 +1,68 @@
 [![Build Status](http://www.molgenis.org/jenkins/buildStatus/icon?job=molgenis)](http://www.molgenis.org/jenkins/job/molgenis/)
+
 # Welcome to MOLGENIS
 
-MOLGENIS is an collaborative open source project on a mission to generate great software infrastructure for life science research. Each app in the MOLGENIS family comes with rich data management interface and plug-in integration of analysis tools in R, Java and web services.
+MOLGENIS is an collaborative open source project on a mission to generate great software infrastructure for life science research.
```
Comment (tommydeboer) [inline README.md]: 'MOLGENIS is a collaborative'

File: README.md [hunk 3]
```diff
@@ -1,84 +1,68 @@
 [![Build Status](http://www.molgenis.org/jenkins/buildStatus/icon?job=molgenis)](http://www.molgenis.org/jenkins/job/molgenis/)
+
 # Welcome to MOLGENIS
 
-MOLGENIS is an collaborative open source project on a mission to generate great software infrastructure for life science research. Each app in the MOLGENIS family comes with rich data management interface and plug-in integration of analysis tools in R, Java and web services.
```
Comment (Mark-de-Haan) [inline README.md]: 'a' collaborative i think? Its a consonant sound

File: README.md [hunk 4]
```diff
@@ -1,84 +1,68 @@
...
-Start Eclipse and select your workspace location (e.g. ~/eclipse/workspace).
+    create database omx;
+    grant all privileges on omx.* to molgenis@localhost identified by 'molgenis';
+    flush privileges;
 
-Click: File -> Import ... -> Existing Maven Projects.
+## 3. Configure Eclipse
 
-Set root directory to your git clone folder. E.g. ~/git (this means you can still see the 'molgenis' folder).
+start Eclipse from the commandline (via Dock will not work!)
```
Comment (tommydeboer) [inline README.md]: Starting via dock doesn't seem to be a problem (any more), at least not on our machines

**Pull request merged**